samples/snippets/fsharp/VS_Snippets_CLR/Array_FindLastIndex/fs/source.fs

```diff
@@ -0,0 +1,44 @@
+﻿// <Snippet1>
+open System
+
+// Search predicate returns true if a string ends in "saurus".
+let endsWithSaurus (s: string) =
+    s.Length > 5 && s.Substring(s.Length - 6).ToLower() = "saurus"
+
+let dinosaurs = 
+    [| "Compsognathus"; "Amargasaurus"
+       "Oviraptor"; "Velociraptor"
+       "Deinonychus"; "Dilophosaurus"
+       "Gallimimus"; "Triceratops" |]
+
+printfn ""
+for dino in dinosaurs do
+    printfn $"{dino}"
+
+Array.FindLastIndex(dinosaurs, endsWithSaurus)
+|> printfn "\nArray.FindLastIndex(dinosaurs, EndsWithSaurus): %i"
+
+Array.FindLastIndex(dinosaurs, 4, endsWithSaurus)
+|> printfn "\nArray.FindLastIndex(dinosaurs, 4, EndsWithSaurus): %i"
+    
+Array.FindLastIndex(dinosaurs, 4, 3, endsWithSaurus)
+|> printfn "\nArray.FindLastIndex(dinosaurs, 4, 3, EndsWithSaurus): %i"
+
+(* This code example produces the following output:

```

Comment:
Any reason why you're using block comments here?  You use line comments later

Reply:
No reason in particular, comments were copied over from the C# snippets, I'll normalize them.